spec.html

```diff
@@ -19008,9 +19051,7 @@ <h1>
           1. If Type(_func_) is not Object, throw a *TypeError* exception.
           1. If IsCallable(_func_) is *false*, throw a *TypeError* exception.
           1. If _tailPosition_ is *true*, perform PrepareForTailCall().
-          1. Let _result_ be Call(_func_, _thisValue_, _argList_).
-          1. Assert: If _result_ is not an abrupt completion, then Type(_result_) is an ECMAScript language type.
```

Comment:
> if completions can only contain language values

This PR has lots of examples of normal completions containing non-language values (e.g., a List, a Record). However, in the "Completion Record Fields" table, `[[Value]]` still says "an ECMAScript language value or ~empty~", so that should probably be fixed.

Reply:
Completion Records can contain things that are not ECMAScript language values. `~unused~` is a very common one. This assertion is removed because it's redundant with the type signature.

Reply:
ah, because `Call` can only return a normal completion when it contains an ES language value, got it. thanks!